covid19-etl/helper/metadata_helper.py

```diff
@@ -62,28 +62,38 @@ def get_existing_data_jhu(self):
         }
 
         print("  summary_report data...")
-        query_string = (
-            '{ summary_report (first: 0, project_id: "'
-            + self.project_id
-            + '") { submitter_id } }'
-        )
-        response = requests.post(
-            "{}/api/v0/submission/graphql".format(self.base_url),
-            json={"query": query_string, "variables": None},
-            headers=self.headers,
-        )
-        assert (
-            response.status_code == 200
-        ), "Unable to query Peregrine for existing 'summary_report' data: {}\n{}".format(
-            response.status_code, response.text
-        )
-        try:
-            query_res = json.loads(response.text)
-        except:
-            print("Peregrine did not return JSON")
-            raise
 
-        for report in query_res["data"]["summary_report"]:
+        summary_reports = []
+        data = None
+        offset = 0
+        first = 10000
```

Comment:
can we make it 100,000, or at least 50,000? i think peregrine can handle it

Reply:
Tested with 100000, works fine. 👍